README.md

```diff
@@ -69,12 +69,20 @@ target = NumericalTarget(
 objective = SingleTargetObjective(target=target)
 ```
 
-In cases where we need to consider multiple (potentially competing) targets, the
-role of the `Objective` is to define additional settings, e.g. how these targets should
-be balanced.
-In `SINGLE` mode, however, there are no additional settings.
-For more details, see 
-[the objective section of the user guide](https://emdgroup.github.io/baybe/userguide/objective.html).
+In situations like the above, where **only one** `Target` is relevant, the role of the
+`Objective` is to merely signal the absence of other `Target`s in the optimization
+problem.  
+**Note:** 

```

Comment:
detail unnecessary for the README and belongs in user guide

Reply:
Move that part (and rewrote slightly to fit into the new context). Resolve if happy